packages/store-ui/src/atoms/Input/Input.stories.mdx

```diff
@@ -0,0 +1,79 @@
+import { Meta, Story, Preview, ArgsTable } from '@storybook/addon-docs/blocks'
+import Input from '.'
+
+<Meta title="Atoms/Input" />
+
+export const colorByState = {
+  default: 'black',
+  success: 'green',
+  error: 'red',
+}
+
+# Default
+
+<Preview>
+  <Story
+    name="Default"
+    args={{
+      placeholder: 'First name',
```

Comment:
Can you add a state via a dropDown in canvas? So we can select the state and see the change
![image](https://user-images.githubusercontent.com/1753396/122989667-760d0380-d379-11eb-87be-ee181866a70f.png)

Reply:
@tlgimenes What do you think about creating only one Input story, then? I was aware of creating that dropdown because doesn't make sense to have the dropdown with the state-variation in the sidebar (Default, Success, Error).

Instead, we can have only one Input story with a dynamic state.

Reply:
Hmmmm. I'm not sure, whatever you prefer